packages/dds/sequence/src/test/intervalCollection.fuzzUtils.ts

```diff
@@ -142,3 +151,51 @@ export function makeReducer(loggingInfo?: LoggingInfo): Reducer<Operation, Clien
 
 	return withLogging(reducer);
 }
+
+const revertibles: IntervalRevertible[] = [];
+
+export function makeRevertibleReducer(
+	loggingInfo?: LoggingInfo,
+): Reducer<RevertOperation, ClientOpState> {
+	const withLogging =
+		<T>(baseReducer: Reducer<T, ClientOpState>): Reducer<T, ClientOpState> =>
+		async (state, operation) => {
+			if (loggingInfo !== undefined) {
+				logCurrentState(state, loggingInfo);
+				console.log("-".repeat(20));
+				console.log("Next operation:", JSON.stringify(operation, undefined, 4));
+			}
+			await baseReducer(state, operation);
+		};
+
+	const reducer = combineReducers<RevertOperation, ClientOpState>({
```

Comment:
i'd avoid duplicating this code. Instead, just add logic to handle revertibles to the existing reducer, but make it configurable whether revertibles get generated. then don't generate them for the current fuzz test.

There are other options here like making the reducer more composable so that you can build on top of it in the revertibles file without duplicating the 'base operations', but I think they're a bit heavy-handed.

Reply:
though, now that I think about it doing the latter option might be a bit cleaner, since it would let you better manage the revertibles queue